index.html

```diff
@@ -136,5 +136,19 @@ <h2 class="about-me-headline">About<br>Myself</h2>
             <p class="under-languages">Skills</p>
             <hr>
         </section>
+
+        <section class="contact-me">
+            <h2 class="contact-me-title">Contact me</h2>
+            <p class="contact-me-description">If you have an application you are interested in developing,
+                 a feature that you need built or a project that needs coding.
+                  I’d love to help with it </p>
+                  <form class="forms" action="https://formspree.io/f/xaykllwd" method="post">
+                    <input class="form-input" type="text" maxlength="30" required> <br>
+                    <input class="form-input" type="email"> <br>
+                    <textarea name="" placeholder="Write your message here" id="" maxlength="500" required></textarea>
+                    <button class="contact-btn">Get In Touch</button>
+                  </form>
```

Comment:
> Great job using FormSpree correctly 💯 However, According to the requirements

- [ ] All fields should have placeholder text, so kindly add placeholder for username and email 👍

- [ ] A user cannot submit the form if one or more of the fields is empty, so kindly add `required` attribute in email field 👍


![1](https://user-images.githubusercontent.com/45231276/199498316-91289e48-6281-49a1-9cb2-0f76583cc049.PNG)


- [ ] When I submit the form I got this screen, So Kindly make sure you have placed the ["name" attribute](https://www.w3schools.com/tags/att_input_name.asp) in all your form elements. 👍


![image](https://user-images.githubusercontent.com/45231276/199498155-03d40dad-1da9-4615-bbf0-4009a717edb6.png) 👍